cartridge/issues.lua

```diff
@@ -91,6 +91,12 @@ local limits_ranges = {
     clock_delta_threshold_warning = {0, math.huge},
 }
 
+local allowed_levels = {
+    critical = true,
+    warning = true,
+    info = true,
+}
+
```

Comment:
I think we could go without this restriction. If a user wants to define a custom level, why would we forbid him